sys/sysmon/include/sysmon/ThreadStatus.hpp

```diff
@@ -47,12 +47,24 @@ class ThreadStatus: public MemoryStatus {
         /// Constructor
         ThreadStatus();
 
+        /// Increment thread loops number
+        inline void inc_loops(void) { loops_++; };
+        /// Increment thread overshots number
+        inline void inc_overshots(void) { overshots_++; };
+        /// Get thread loops number
+        inline uint32_t loops() const { return loops_; };
```

Comment:
Your are using either `void f(void)` or `void f()`. It means the same in C++.
I prefer `void f()` which is used in almost all C++ libraries.

Reply:
done... painful too :)